### PR TITLE
Merging to Synchronize + cleanup

### PR DESCRIPTION
**Pull request by ngupta23**
merged local changes with github develop plus some cleanup based on linter recommendations.

### CODE REVIEW

**Comment by AutoViML** [comment]
Please go ahead with the merge.

**Comment by ngupta23** [comment]
Hi @AutoViML 

I can not merge it until you approve it. You should see an option to do so in the pull request.

Thanks!

**Review by AutoViML — APPROVED** [review]

**Comment by AutoViML** [comment]
Ok successfully merged #22
Ram

**Pull request merged**
